tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/CallRetry.java

```diff
@@ -50,23 +51,33 @@ public CallRetry(Callable<R> mainAttempt, Callable<Optional<R>> fallbackAttempt)
      * @return Results of main call, or fallback result if data is present after main failure
      */
     public R attemptWithRetry(int maxTries) {
+        log.info("attempt with retry start {}", new Date().toString());
```

Comment:
Any particular reason for all the `new Date().toString()` logging? The log line already has the date time.

Reply:
Removed all unnecessary logs.